### PR TITLE
Changed the translation

### PR DESCRIPTION
**Pull request by Tirokk**
> [Ruben80](https://github.com/Ruben80) **Authored by [Ruben80](https://github.com/Ruben80)**
_<time datetime="2020-02-20T20:19:47Z" title="Thursday, February 20th 2020, 9:19:47 pm +01:00">Feb 20, 2020</time>_
_Closed <time datetime="2020-03-25T08:51:05Z" title="Wednesday, March 25th 2020, 9:51:05 am +01:00">Mar 25, 2020</time>_
---

Did the following:

Rename "follow" (and similar occurrences) to "subscribe" in the code
Change the translations from "follow" and "folgen" to "subscribe" and "abonnieren"

The last one I did not know what to do: Rename the FOLLOW relationship in the database and write a migration to reflect that change

### Issues

- None

### Todo

- [X] None

### CODE REVIEW

File: webapp/locales/de.json [hunk 1]
```diff
@@ -351,8 +351,8 @@
     }
   },
   "followButton": {
-    "follow": "Folgen",
-    "following": "Folge Ich"
+    "follow": "Abonnieren",
+    "following": "Abonniere Mich"
```
Comment (Tirokk) [inline webapp/locales/de.json:355]: > [Mogge](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-02-21T11:05:46Z" title="Friday, February 21st 2020, 12:05:46 pm +01:00">Feb 21, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -351,8 +351,8 @@
     }
   },
   "followButton": {
-    "follow": "Folgen",
-    "following": "Folge Ich"
+    "follow": "Abonnieren",
+    "following": "Abonniere Mich"
```


This sounds strange. May bie just `Abonniert`?

File: webapp/locales/de.json [hunk 2]
```diff
@@ -498,9 +498,9 @@
   },
   "profile": {
     "commented": "Kommentiert",
-    "follow": "Folgen",
-    "followers": "Folgen",
-    "following": "Folge Ich",
+    "follow": "Abonnieren",
+    "followers": "Abonnieren",
```
Comment (Tirokk) [inline webapp/locales/de.json:502]: > [Mogge](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-02-21T11:23:32Z" title="Friday, February 21st 2020, 12:23:32 pm +01:00">Feb 21, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -498,9 +498,9 @@
   },
   "profile": {
     "commented": "Kommentiert",
-    "follow": "Folgen",
-    "followers": "Folgen",
-    "following": "Folge Ich",
+    "follow": "Abonnieren",
+    "followers": "Abonnieren",
```


Once again, this sounds strange. `Wird abonniert von` ?

File: webapp/locales/de.json [hunk 3]
```diff
@@ -498,9 +498,9 @@
...
+    "following": "Auf Abonniert",
```
Comment (Tirokk) [inline webapp/locales/de.json:503]: > [Mogge](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-02-21T11:24:02Z" title="Friday, February 21st 2020, 12:24:02 pm +01:00">Feb 21, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -498,9 +498,9 @@
   },
   "profile": {
     "commented": "Kommentiert",
-    "follow": "Folgen",
-    "followers": "Folgen",
-    "following": "Folge Ich",
+    "follow": "Abonnieren",
+    "followers": "Abonnieren",
+    "following": "Auf Abonniert",
```


`Hat abonniert` ?

File: webapp/locales/de.json [hunk 4]
```diff
@@ -510,10 +510,10 @@
...
     "network": {
       "andMore": "und {number} weitere …",
-      "followedBy": "wird gefolgt von:",
-      "followedByNobody": "wird von niemandem gefolgt.",
-      "following": "folgt:",
-      "followingNobody": "folgt niemandem.",
+      "followedBy": "wird abboniert von:",
+      "followedByNobody": "wird von niemandem abonniert.",
+      "following": "abonniert:",
+      "followingNobody": "abboniert auf niemandem.",
```
Comment (Tirokk) [inline webapp/locales/de.json:516]: > [Mogge](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-02-21T11:28:50Z" title="Friday, February 21st 2020, 12:28:50 pm +01:00">Feb 21, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -510,10 +510,10 @@
     "name": "Mein Profil",
     "network": {
       "andMore": "und {number} weitere …",
-      "followedBy": "wird gefolgt von:",
-      "followedByNobody": "wird von niemandem gefolgt.",
-      "following": "folgt:",
-      "followingNobody": "folgt niemandem.",
+      "followedBy": "wird abboniert von:",
+      "followedByNobody": "wird von niemandem abonniert.",
+      "following": "abonniert:",
+      "followingNobody": "abboniert auf niemandem.",
```


`abonniert niemanden`

**Comment by Tirokk** [comment]
> [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-03-25T08:51:05Z" title="Wednesday, March 25th 2020, 9:51:05 am +01:00">Mar 25, 2020</time>_

---


closing this as we've been waiting more than a month for a response from @Ruben80 
and it's from a `fork` so there is no way for it to pass the build/get merged. See :point_up_2:

**Review by Mogge — CHANGES_REQUESTED** [review]
hey @Ruben80  can you please put in a PR directly to the parent repo?
this won't pass the build cause we have encrypted build variables and they are not exposed to forks for security reasons. See https://docs.travis-ci.com/user/environment-variables/#defining-encrypted-variables-in-travisyml

**Review by Mogge — CHANGES_REQUESTED** [review]
I am not that happy with that change in the German translations. The word `abonniert` sounds very technical.

**Pull request closed without merging**